graphql/version_resolver.go

```diff
@@ -149,6 +149,46 @@ func (r *versionResolver) ChildVersions(ctx context.Context, obj *restModel.APIV
 	return nil, nil
 }
 
+// ExternalLinksForMetadata is the resolver for the externalLinksForMetadata field.
+func (r *versionResolver) ExternalLinksForMetadata(ctx context.Context, obj *restModel.APIVersion) ([]*ExternalLinkForMetadata, error) {
+	pRef, err := data.FindProjectById(*obj.Project, false, false)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error finding project `%s`: %s", *obj.Project, err.Error()))
+	}
+	if pRef == nil {
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("Project `%s` not found", *obj.Project))
+	}
+	var externalLinks []*ExternalLinkForMetadata
+	for _, link := range pRef.ExternalLinks {
+		// replace {version_id} with the actual version id
+		formattedURL := strings.Replace(link.URLTemplate, "{version_id}", *obj.Id, -1)
+		externalLinks = append(externalLinks, &ExternalLinkForMetadata{
+			URL:         formattedURL,
+			DisplayName: link.DisplayName,
+		})
+	}
+	return externalLinks, nil
+}
+
+// GitTags is the resolver for the gitTags field.
+func (r *versionResolver) GitTags(ctx context.Context, obj *restModel.APIVersion) ([]*GitTag, error) {
+	v, err := model.VersionFindOneId(utility.FromStringPtr(obj.Id))
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding version with id `%s`: %s", *obj.Id, err.Error()))
+	}
+	if v == nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding version with id `%s`: %s", *obj.Id, "version not found"))
+	}
+	var gitTags []*GitTag
+	for _, tag := range v.GitTags {
```

Comment:
since GitTags is already part of the original version object and not in a different collection, why don't we get it from obj so that we don't need an additional db call?

Reply:
Sure I could do that, but I think it would require exposing the git tags via the rest route and I wasn't sure if that's desirable or not. I added a commit that represents this work!

Reply:
Since the end result is the data being exposed to the user along with the patch without any special restrictions, I don't personally see a downside to adding it to the restAPI.  

We may want to expand the version graphql tests to query for the tag field as well.